lib/transloadit/assembly.rb

```diff
@@ -1,35 +1,12 @@
 require 'transloadit'
 
 #
-# Represents a Assembly ready to be sent to the REST API for processing. An
-# Assembly can contain one or more Steps for processing or point to a
-# server-side template. It's submitted along with a list of files to process,
-# at which point Transloadit will process and store the files according to the
-# rules in the Assembly.
+# Represents an Assembly API ready to make calls to the REST API endpoints.
 #
-# See the Transloadit {documentation}[http://transloadit.com/docs/building-assembly-instructions]
-# for futher information on Assemblies and their parameters.
+# See the Transloadit {documentation}[https://transloadit.com/docs/api-docs/#assembly-api]
+# for futher information on Assemblies and available endpoints.
 #
-class Transloadit::Assembly
-  # @return [Transloadit] the associated Transloadit instance
-  attr_reader   :transloadit
-
-  # @return [Hash] the options describing the Assembly
-  attr_accessor :options
-
-  #
-  # Creates a new Assembly authenticated using the given +transloadit+
-  # instance.
-  #
-  # @param [Transloadit] transloadit the associated Transloadit instance
-  # @param [Hash]        options     the configuration for the Assembly;
-  #   see {Transloadit#assembly}
-  #
-  def initialize(transloadit, options = {})
-    self.transloadit = transloadit
-    self.options     = options
-  end
-
+class Transloadit::Assembly < Transloadit::API
```

Comment:
Not sure if "API" is a good word here. You want to name a parent for both "Assembly" and "Template" correct? How about "TransloaditApiObject" or "ApiObject"? "API" is too general I feel.